### PR TITLE
Discourse v3

### PR DESCRIPTION
**Pull request by Salamandar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by Salamandar** [comment]
It now requires Imagemagick 7 for the `magick` command line tool…

https://github.com/discourse/discourse_docker/blob/85a6e38644be8baf9820e2b37f5c3d8a8e39955b/image/base/install-imagemagick#L5

File: conf/nginx.conf [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 #sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
   location __PATH__/ {
-    alias __INSTALL_DIR__/public/ ;
+    alias __INSTALL_DIR__/discourse/ublic/ ;
```
Comment (ericgaspar) [inline conf/nginx.conf:32]: `alias __INSTALL_DIR__/discourse/public/ ;`

Comment (Salamandar) [reply conf/nginx.conf:32]: gniiiii thanks

**Pull request merged**